packages/console/src/components/ImagePlaceholder/index.tsx

```diff
@@ -4,8 +4,17 @@ import defaultPlaceholder from '@/assets/images/default-placeholder.svg';
 
 import * as styles from './index.module.scss';
 
-const ImagePlaceholder = () => {
-  return <img src={defaultPlaceholder} className={styles.placeholder} />;
+type Props = {
+  size?: number;
+  borderRadius?: number;
+};
+
+const ImagePlaceholder = ({ size = 50, borderRadius = 8 }: Props) => {
+  return (
+    <div className={styles.container} style={{ width: size, height: size, borderRadius }}>
+      <img src={defaultPlaceholder} />
```

Comment:
nit: giving img an alt attribute would be great.

Reply:
right